### DISCUSSION TITLE
Feature request: termux.properties extra keyboard combos

### DISCUSSION THREAD

**Discussion by nathaneltitane** [termux/termux-app#2569] [Ideas]
I do not know if this has surfaced in a ny of the feature requests in the bug tracker, but using Dex, there can be times where manipulating the device directly can be very umbersome, as it is docked.

Implementing the addition of extra keuboard combo keys to certain termus features, such as the toggle for the bottom bar would be very much appreciated, or at the very least, the detection of physical keyboard input to emulate the already existing combo (through bluetooth)

this is what happens when a session is already running and termux is shown on Dex/external monitor:

![Screenshot_20220206-142345_Termux.jpg](https://user-images.githubusercontent.com/5674757/152697974-6e96d3a7-244a-486b-9b77-39cd9fa33469.jpg)

**Comment by nathaneltitane** [comment]
Anyone?

**Reply by agnostic-apollo** [reply]
What you are asking will likely take considerable time to implement and won't be done by me anytime soon, lot of other work to be done. Send pull request if possible.

**Reply by nathaneltitane** [reply]
I would, but unfortunately I don't Java... Will wait, this is just a nice thing to consider implementing for polishing the experience. Thanks for your reply.
